e2e/ephemeral-peer.dockerfile

```diff
@@ -0,0 +1,20 @@
+# syntax=docker/dockerfile:1.2
+# Nb: the "latest" tag needs to be managed manually, CI doesn't update it
+FROM gcr.io/opensourcecoin/radicle-link-seedling-build:latest AS build
+WORKDIR /build
+RUN --mount=type=bind,source=.,target=/build,rw \
```

Comment:
You know what's really taking all the fun out of this? If we _don't_ add `target/` to `.dockerignore`, `docker build` will send like 60GiB to the doge daemon. _However_, it apparently also respects `.dockerignore` when mounting, so we never see any caching.